docs/markdown/Using Pants/concepts/targets.md

```diff
@@ -154,6 +154,52 @@ You only need to declare direct dependencies. Pants will pull in _transitive dep
 > 
 > Transitive excludes can only be used in target types that conventionally are not dependend upon by other targets, such as `pex_binary` and `python_test` / `python_tests`. This is meant to limit confusion, as using `!!` in something like a `python_source` / `python_sources` target could result in surprising behavior for everything that depends on it. (Pants will print a helpful error when using `!!` when it's not legal.)
 
+Field default values
+====================
+
+As mentioned above in [BUILD files](doc:targets#build-files), most fields use sensible defaults. And
+for specific cases it is easy to provide some other value to a specific target. The issue is if you
+want to apply a specific non-default value for a field on many targets. This can get unwieldy, error
+prone and hard to maintain. Enter `__defaults__`.
+
+Default field values per target are set using the `__defaults__` BUILD file symbol, and apply to the
+current subtree.
+
+Default fields and values are validated against their target types, except when provided using the
+`all` keyword, in which case only values for fields applicable to each target are validated.
+
+This means, that it is legal to provide a default value for `all` targets, even if it is only a
+subset of targets that actually supports that particular field.
+
+Examples:
+
+```python src/example/BUILD
+    # Provide default `tags` to all targets in this subtree, and skip black, where applicable.
+    __defaults__(all=dict(tags=["example"], skip_black=True))
+```
+
+Subdirectories may override defaults from a parent BUILD file:
+
+```python src/example/override/BUILD
+    # For `files` and `resources` targets, we want to use some other defaults.
+    __defaults__({
```

Comment:
Could you add a little bit of prose explaining that, unlike `all` which is a keyword arg, we need to pass a dictionary as a positional arg to update the defaults for specific targets?

Reply:
Yeah, that makes sense.